packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -66,6 +66,15 @@ export type AutocompleteProps<
     undefined,
     IsCustomValueAllowed
   >["defaultValue"];
+  /**
+   * The default value. Use when the component is not controlled.
+   */
+  getOptionLabel?: UseAutocompleteProps<
+    OptionType,
+    HasMultipleChoices,
+    undefined,
+    IsCustomValueAllowed
+  >["getOptionLabel"];
```

Comment:
What's the `undefined` used for? It's `undefined` in a few other places, but typically, you want generics like that to be passed-in.